Refer to this link for build results (access rights to CI server needed): 
http://bock.systems:8080/job/hangups_tox/21/

**Build Log**
_last 25 lines_

```
[...truncated 8 lines...]
 > git -c core.askpass=true fetch --tags --progress https://github.com/TheInnerCircleO/hangupsbot +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/16/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/16/merge^{commit} # timeout=10
Checking out Revision be7e5d86c8770baa25e68435e9432470e131cc40 (refs/remotes/origin/pr/16/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f be7e5d86c8770baa25e68435e9432470e131cc40
First time build. Skipping changelog.
[workspace] $ /bin/bash /tmp/hudson4885241598160297720.sh
pep8 recreate: /var/lib/jenkins/jobs/hangups_tox/workspace/.tox/pep8
pep8 installdeps: -r/var/lib/jenkins/jobs/hangups_tox/workspace/requirements.txt, -r/var/lib/jenkins/jobs/hangups_tox/workspace/test-requirements.txt
pep8 develop-inst: /var/lib/jenkins/jobs/hangups_tox/workspace
pep8 runtests: PYTHONHASHSEED='1237644907'
pep8 runtests: commands[0] | flake8
./hangupsbot/commands.py:489:24: W291 trailing whitespace
                i['t'],
                       ^
./hangupsbot/commands.py:495:80: E501 line too long (91 > 79 characters)
            result.append(hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK))
                                                                               ^
ERROR: InvocationError: '/var/lib/jenkins/jobs/hangups_tox/workspace/.tox/pep8/bin/flake8'
___________________________________ summary ____________________________________
ERROR:   pep8: commands failed
Build step 'Execute shell' marked build as failure
Setting status of 1a96086918f1f108f38dd0f3ddca7edd338cb4bb to FAILURE with url http://bock.systems:8080/job/hangups_tox/21/ and message: Merged build finished.
```

Oh No...

--- a/hangupsbot/commands.py
+++ b/hangupsbot/commands.py
@@ -491,7 +491,12 @@ def stock(bot, event, *args):
                 link_target="https://www.google.com/finance?q=" + i['t']
             )
             result.append(ticker_link)
-            result.append(': ' + i['l'] + ' (' + i['cp'] + '%)')
+            #result.append(': ' + i['l'] + ' (' + i['cp'] + '%)')
+            results.append(
+                hangups.ChatMessageSegment(
+                    ': {} | {} ({}%)'.format(i['l'], i['c'], i['cp'])
+                )
+            )
             result.append(
                 hangups.ChatMessageSegment(
                     '\n',

--- a/hangupsbot/commands.py
+++ b/hangupsbot/commands.py
@@ -486,7 +486,7 @@ def stock(bot, event, *args):
         data = json.loads(raw_data.text[3:])
         for i in data:
             ticker_link = hangups.ChatMessageSegment(
-                i['t'], 
+                i['t'],
                 hangups.SegmentType.LINK,
                 link_target="https://www.google.com/finance?q=" + i['t']
             )

--- a/hangupsbot/commands.py
+++ b/hangupsbot/commands.py
@@ -492,7 +492,12 @@ def stock(bot, event, *args):
             )
             result.append(ticker_link)
             result.append(': ' + i['l'] + ' (' + i['cp'] + '%)')
-            result.append(hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK))
+            result.append(
+                hangups.ChatMessageSegment(
+                    '\n',
+                    hangups.SegmentType.LINE_BREAK
+                )
+            )
     except:
         result = "Whatchoo tryna do? check urself."
 

